ql/src/java/org/apache/hadoop/hive/ql/exec/repl/util/ReplUtils.java

```diff
@@ -219,16 +224,16 @@
     return TaskFactory.get(replLogWork, conf);
   }
 
+  public static boolean isDbBeingFailedOverAtSource(Database db) {
+    assert (db != null);
+    Map<String, String> dbParameters = db.getParameters();
+    return Failover_Point.SOURCE.toString().equalsIgnoreCase(dbParameters.get(ReplConst.REPL_FAILOVER_ENABLED));
```

Comment:
db.getParameters() is Nullable, don't you require a null check like you did in other places?

Reply:
May be we can just have a single method like this?
public static boolean isDbBeingFailedOverAtEndpoint(Database db, Failover_Point endPoint) {
    assert (db != null);
    Map<String, String> dbParameters = db.getParameters();
    return endPoint.toString().equalsIgnoreCase(dbParameters.get(ReplConst.REPL_FAILOVER_ENABLED));
  }

Reply:
Also, ReplConst.REPL_FAILOVER_ENABLED -> ReplConst.REPL_FAILOVER_ENDPOINT
Does this make sense?

Reply:
Refractored.